index.html

```diff
@@ -38,150 +38,36 @@ <h1>
-            <div class="grid">
-                <div class="grid-item">
-                    <h2>Multi-Post Stories Gain+Glory</h2>
-                </div>
-                <div class="grid-item">
-                    <ul>
-                        <li> Ruby on Rails</li>
-                        <li> CSS </li>
-                        <li> JavaScript </li>
-                        <li> HTML </li>
-                    </ul>
-                </div>
-                <div class="grid-item">
-                    <button class="normal-btn">
-                        See Project
-                    </button>
-                </div>
-            </div>
-        </div>
-        <div class="background">
-            <div class="grid">
-                <div class="grid-item">
-                    <h2>Multi-Post Stories Gain+Glory</h2>
-                </div>
-                <div class="grid-item">
-                    <ul>
-                        <li> Ruby on Rails</li>
-                        <li> CSS </li>
-                        <li> JavaScript </li>
-                        <li> HTML </li>
-                    </ul>
-                </div>
-                <div class="grid-item">
-                    <button class="normal-btn">
-                        See Project
-                    </button>
-                </div>
-            </div>
-        </div>
-        <div class="background">
-            <div class="grid">
-                <div class="grid-item">
-                    <h2>Multi-Post Stories Gain+Glory</h2>
-                </div>
-                <div class="grid-item">
-                    <ul>
-                        <li> Ruby on Rails</li>
-                        <li> CSS </li>
-                        <li> JavaScript </li>
-                        <li> HTML </li>
-                    </ul>
-                </div>
-                <div class="grid-item">
-                    <button class="normal-btn">
-                        See Project
-                    </button>
-                </div>
-            </div>
-        </div>
-        <div class="background">
-            <div class="grid">
-                <div class="grid-item">
-                    <h2>Multi-Post Stories Gain+Glory</h2>
-                </div>
-                <div class="grid-item">
-                    <ul>
-                        <li> Ruby on Rails</li>
-                        <li> CSS </li>
-                        <li> JavaScript </li>
-                        <li> HTML </li>
-                    </ul>
-                </div>
-                <div class="grid-item">
-                    <button class="normal-btn">
-                        See Project
-                    </button>
-                </div>
-            </div>
-        </div>
-    </div>
+    
+    <section class="portfolio" id="portfolio">
+        <h2>My Recent Works</h2> 
     </section>
+
+    <div class="works">
+    
+    </div>
+
+    <div class="project-pop">
+            
```

Comment:
[OPTIONAL]
- [ ] I kindly suggest that you follow the Figma design in making your modals
- Reduce the height of your button like skills list to match the Figma design.
- The button should appear to the right of the modal


|  |  |
| ------------- | ------------- |
| Your Modal | <img width="707" alt="image" src="https://github.com/Arch-Noize/portfolio-microverse/assets/52934128/f80a4041-8f9e-4600-ab50-bf6d533b574e"> |
| Figma Design Modal  | <img width="412" alt="image" src="https://github.com/Arch-Noize/portfolio-microverse/assets/52934128/953fa3df-93f1-4c06-acdd-dfd14c19b11c">  |